volume/ceph/ceph.go

```diff
@@ -204,6 +253,19 @@ func (v *Volume) Unmount(id string) error {
 	if err := v.release(); err != nil {
 		return err
 	}
+
+	fsType, err := utils.DeviceHasFilesystem(v.mappedDevicePath)
+	if err != nil {
+		return err
+	}
+
+	if fsType == "crypto_LUKS" {
```

Comment:
Fixed in https://github.com/medallia/docker/pull/15/commits/975c9c2ab2e543bd69892b3254de4e3e90c1156e